source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseWalletDialog.js

```diff
@@ -0,0 +1,178 @@
+// @flow
+import React, { Component } from 'react';
+import { defineMessages, intlShape, FormattedHTMLMessage } from 'react-intl';
+import { Select } from 'react-polymorph/lib/components/Select';
+import { SelectSkin } from 'react-polymorph/lib/skins/simple/SelectSkin';
+import { Stepper } from 'react-polymorph/lib/components/Stepper';
+import { StepperSkin } from 'react-polymorph/lib/skins/simple/StepperSkin';
+import styles from './DelegationStepsChooseWalletDialog.scss';
+import DialogCloseButton from '../../widgets/DialogCloseButton';
+import DialogBackButton from '../../widgets/DialogBackButton';
+import Dialog from '../../widgets/Dialog';
+
+const messages = defineMessages({
+  title: {
+    id: 'delegation.setup.steps.dialog.title',
```

Comment:
Please, add `staking.` to all INTL ids (e.g. `staking.delegation.setup.steps.dialog.title`) so we have everything related to Staking in the same place.